Makefile

```diff
@@ -36,13 +33,10 @@ clean:
 	go clean -i ./...
 	rm -rf build
 
-.PHONY: vet
```

Comment:
I would still run `go vet`

Reply:
`go vet` is included in golangci-lint (see https://golangci-lint.run/usage/linters/). Is there any benefit to run it separately?